app/util/analytics.py

```diff
@@ -134,26 +134,16 @@ def get_test_count_by_type(tests_type, log):
                     tests[test_name] = test_rate
         return tests
 
-    @staticmethod
-    def get_success_count_from_tests(tests):
-        success_test_count = 0
-        for success_rate in tests.values():
-            if success_rate >= SUCCESS_TEST_RATE:
-                success_test_count = success_test_count + 1
-        return success_test_count
-
     def set_actual_test_count(self):
-        TEST_RESULTS_START_STRING = 'Request label stats:'
-        res_string_idx = [index for index, value in enumerate(self.bzt_log_file) if TEST_RESULTS_START_STRING in value]
+        test_result_string_trigger = 'Request label stats:'
+        res_string_idx = [index for index, value in enumerate(self.bzt_log_file) if test_result_string_trigger in value]
         # Cut bzt.log from the 'Request label stats:' string to the end
         if res_string_idx:
             res_string_idx = res_string_idx[0]
             results_bzt_run = self.bzt_log_file[res_string_idx:]
 
-            selenium_tests = self.get_test_count_by_type(tests_type='selenium', log=results_bzt_run)
-            jmeter_tests = self.get_test_count_by_type(tests_type='jmeter', log=results_bzt_run)
-            self.selenium_test_count = self.get_success_count_from_tests(selenium_tests)
-            self.jmeter_test_count = self.get_success_count_from_tests(jmeter_tests)
+            self.selenium_test_count = self.get_test_count_by_type(tests_type='selenium', log=results_bzt_run)
```

Comment:
seems like it is not `test_count` it is not a count at all. It is a dictionary where the key is test name and value is test success rate. Suggest change variable and function name to something that contains words: `test_rates` or `test_run_rates` or `rate_by_test_names` Please change all related variables and functions.